t/tasks/inventory/macos/hostname.t

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use lib 't/lib';
+
+use Test::Deep;
+use Test::Exception;
+use Test::More;
+use Test::NoWarnings;
+
+use FusionInventory::Agent::Task::Inventory::MacOS::Hostname;
+
+my %tests = (
+    'hostname' => 'mac-fusion01'
```

Comment:
You should better use filename of file to use from resources/macos/system_profiler as key, so you can plan test with counted keys and make a foreach loop on each key. Do that even if you have one test as this make your test ready to accept new system_profile samples.

You can check other unittest script doing same things